crypto/aes/asm/aesni-x86_64.pl

```diff
@@ -986,6 +986,7 @@ sub aesni_generate8 {
 .type	aesni_ccm64_encrypt_blocks,\@function,6
 .align	16
 aesni_ccm64_encrypt_blocks:
+.cfi_startproc
 ___
 $code.=<<___ if ($win64);
 	lea	-0x58(%rsp),%rsp
```

Comment:
Should we do something with the CFA here?

Reply:
no, I can't do anything for win64

Reply:
win64 unwind info is totally broken, I wonder if it would
be better to remove that altogether, @davidben ?

Reply:
I missed the win64 part, but saw the win64 part in one of the other PRs.

Reply:
as far as I can tell the .cfi directives are ignored by windows.
The unwinding cannot not work correctly when this block is interrupted.
It appears to be correcting the stack and restoring xmm6-9 only after
.Lccm64_enc_body:
but when interrupted directly after the lea only the stack needs to be corrected,
but the registers are not yet saved.
I wonder how that stuff was tested.